.mergify.yml

```diff
@@ -22,10 +22,10 @@ pull_request_rules:
       update:
   - name: merge scala-steward's PRs
     conditions:
-      - author=scala-steward
+      - author=dwolla-oss-scala-steward[bot]
       - status-success=Build and Test (ubuntu-latest, 2.13.6, adopt@1.8)
```

Comment:
Could you please change `2.13.6` to `2.13.8` throughout this file? It looks like we missed that change in #48 and Mergify won't see future builds as successes without it.